---
layout: archive
title: "CV"
permalink: /cv/
author_profile: true
redirect_from:
  - /resume
---

{% include base_path %}
[Download CV here](../files/Tu_B_CV_2019.pdf)
Education
======
* B.S. in Biochemistry, University of Nebraska--Lincoln, 2015


Research experience
======
Undergraduate Student
University of Nebraska--Lincoln    October 2012--February 2016
Advisors: Concetta C. DiRusso and Paul N. Black

Project 1
------
Transcriptional response of Chlamydomonas reinhardtii to small lipid-inducing molecules
* Conceived the project, produced RNA samples, compared different bioinformatic tools and performed statistically sound data analysis

Project 2
------
High-throughput phenotypic screens for lipid-inducing small molecules in green algae
* Performed high-throughput screens using automated liquid handling system and optimized protocols

Project 3
------
Quantification of lipid metabolism-related gene expression in Chlamydomonas reinhardtii
* Adapted and optimized RNA isolation and qPCR methods for algae

Project 4
------
Exogenous Citrate Induces Lipid Accumulation in Chlamydomonas reinhardtii
* Conceived the project, independently designed and carried out experiments on lipid and metabolite quantification
  
Skills
======
* Research techniques: transcriptomics, functional genomic analysis, data visualization, high-throughput phenotypic screen, qPCR, GC/MS metabolite quantification, NMR structural analysis, automated liquid handling system, gel electrophoresis, protein purification and enzyme assays
* Technical strengths: 
	* Programming languages: Python, C++, Java, R
	* Operating systems: Linux, Windows, MacOS
	* Parallel programming environments: OpenMP, MPI, CUDA
	* Typesetting and Web content: LaTeX, HTML, CSS
	* Cloud and cluster computing: AWS, Slurm

Publications
======
  <ul>{% for post in site.publications %}
    {% include archive-single-cv.html %}
  {% endfor %}</ul>
  
Talks
======
  <ul>{% for post in site.talks %}
    {% include archive-single-talk-cv.html %}
  {% endfor %}</ul>
  

